upload-to-release

```diff
@@ -22,21 +25,20 @@ if [ "$IS_DRAFT" = true ]; then
   exit 0
 fi
 
+# Expand the filename to allow for glob pattern
+FILENAME=$(basename $1)
```

Comment:
This could end up being multiple files - we'd either need to handle that, or bail if the glob returns multiple files. I'd opt for bailing on multiple matches, I think it would lead to fewer accidental uploads.

Reply:
Since the API supports both I could add multi files if need be.

Reply:
Ooh that's true. I'm a little worried that that'd make for unintentional uploads though, but I guess that's up to the people writing the glob.

Reply:
I can have it print out the files before uploading? At least then the user would know what's been uploaded.

Reply:
Yeah I think that's a great idea!